OutOfSchool/OutOfSchool.WebApi.Tests/Services/ChildServiceTests.cs

```diff
@@ -260,37 +260,81 @@ public async Task GetApprovedByWorkshopId_NoExistingChildren_ShouldReturnEmptySe
     //        Assert.ThrowsAsync<ArgumentException>(() => childService.UpdateChildForParetWithSpecifiedUserId(child));
     //    }
 
-    //    [Test]
-    //    public void ChildService_Delete_DeletesChild()
-    //    {
-    //        ChildDto child = new ChildDto { Id = 1, FirstName = "fn1", LastName = "ln1", MiddleName = "mn1", DateOfBirth = new DateTime(2003, 11, 9), Gender = Gender.Male, ParentId = 1, SocialGroupId = 2 };
-    //        mockRepository.Setup(m => m.Delete(It.IsAny<Child>()));
-    //        mockRepository.Setup(m => m.GetById(It.IsAny<long>())).Returns(Task.FromResult(GetTestChildEntities().First()));
+    [Test]
+    public async Task ChildService_DeleteWithTechAdminRole_DeletesChild()
+    {
+        // Arrange
+        var child = new Child()
+        {
+            Id = Guid.NewGuid(),
+            Parent = new Parent() { UserId = Guid.NewGuid().ToString() },
+        };
```

Comment:
(Fake test data) Please consider using `Bogus` library for such operation. 

It could be as simple as `Faker<TYourType>.Generate()` but for this specific case we already have `ChildGenerator` extension class that could be either used as is
`    var child = ChildGenerator.Generate().WithParent(ParentGenerator.Generate());`
or enhanced with new simple method `WithGeneratedParent() => WithParent(ParentGenerator.Generate())` that slightly decrease test code complexity like
`    var child = ChildGenerator.Generate().WithGeneratedParent();`

Reply:
Fixed